bee-hive/bee_hive/__init__.py

```diff
@@ -1,6 +1,3 @@
-
-__all__ = [
-    "Agent",
-    "Workflow",
-    "Interface",
-]
+# SPDX-License-Identifier: Apache-2.0
+from .workflow import Workflow
+from .bee_agent import BeeAgent
```

Comment:
@akihikokuroda  is this what you'd need?

```suggestion
from .bee_agent import BeeAgent
from .agent import save_agent, restore_agent, remove_agent
```

Reply:
Yes, it is.

Reply:
Thanks for clarifying 
@psschwei I had force-pushed after george's rebase before I saw this update.
I've reinstated the change so we should be good